exercises/SS22/ex_rnw/sol_Ex_1.Rnw

```diff
@@ -0,0 +1,202 @@
+
+\begin{enumerate}
+
+  \item 
+  
+  For the optimal constant model $\fx = \bm{\theta}$ for the loss  $\Lxy=  \big(m(y)-m(\fx)\big)^2,$ we first apply the following substitution $z^{(i)} = m(\yi)$ for each $i=1,\ldots,n,$ and introduce $\bm{\theta}_m = m(\bm{\theta}) \in m(\R).$ 
+%  
+ Note that the inverse of $m$ is continuous and strictly monotone as well, so that the minimizer of the initial optimization problem, i.e.,
+%
+$$
+\min_{f \in \Hspace} \riskef =  \min_{\bm{\theta} \in \R} \sumin ( m(\yi) - m(\bm{\theta}))^2. 
+$$
+%
+is the same as for the ``substituted'' optimization problem, i.e.,
+%
+%
+$$
+m^{-1} \left( \min_{\bm{\theta}_m \in  m(\R)} \sumin (z^{(i)} - \bm{\theta}_m )^2\right) .
+$$
+%
+For the latter we have seen in the lecture (optimizer of the empirical L2 risk) that
```

Comment:
mit "latter" meinst Du das in der Klammer, oder?

Reply:
Ja, habe ich jetzt nochmal etwas expliziter formuliert